### PR TITLE
Create event

### PR DESCRIPTION
**Pull request by donikrizky**

### CODE REVIEW

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java [hunk 1]
```diff
@@ -37,7 +38,7 @@ public EventServiceImpl(EventRepository eventRepository, UserRepository userRepo
 	@Override
 	public Event create(Long userId, CreateEventWrapper wrapper) {
 		User user = userRepository.findById(userId)
-				.orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Error: User not found!"));
+				.orElseThrow(() -> new ResourceNotFoundException("User", "id", userId));
 
 		if (wrapper.getMaximumAge() > 40 || wrapper.getMinimumAge() < 18) {
 			throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Error: Age must be between 18 and 40!");
```
Comment (akbarb24) [inline src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java]: I think it should throws new BadRequestException

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java [hunk 2]
```diff
@@ -87,7 +80,8 @@ public Event create(Long userId, CreateEventWrapper wrapper) {
 		} else if (direction.equalsIgnoreCase("ASC")) {
 			paging = PageRequest.of(pageNumber, pageSize, Sort.by(sortBy).ascending());
 		} else {
-			throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Direction error!");
+			throw new ResponseStatusException(HttpStatus.BAD_REQUEST,
```
Comment (akbarb24) [inline src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java]: I think it should throws new BadRequestException

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java [hunk 3]
```diff
@@ -95,30 +97,28 @@ public Event create(Long userId, CreateEventWrapper wrapper) {
 	@Override
 	public Event edit(Long userId, EditEventWrapper wrapper) {
 		Event event = eventRepository.findById(wrapper.getEventId())
-				.orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Error: Event not found!"));
+				.orElseThrow(() -> new ResourceNotFoundException("Event", "id", wrapper.getEventId()));
 
-		if(!event.getUser().getUserId().equals(userId)) {
-			throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, "Error: Users are not authorized to edit this event");
+		if (!event.getUser().getUserId().equals(userId)) {
+			throw new ResponseStatusException(HttpStatus.UNAUTHORIZED,
```
Comment (akbarb24) [inline src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java:103]: IDK, If we throw ResponseStatusException, does the exception handler in controller catch it? Have u test it?

Comment (donikrizky) [reply src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java:103]: it should be, I have tried it with another ResponseStatusException but different HttpStatus, but again, the HttpStatus shouldn't be concerned

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java (hunk 2 above)
Comment (donikrizky) [reply src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java]: oh yeah forgot haha

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java (hunk 1 above)
Comment (donikrizky) [reply src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java]: I copied from the UserService before, is it changed now?

Comment (akbarb24) [reply src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java]: Yep.

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java (hunk 3 above)
Comment (akbarb24) [reply src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java:103]: Oo okay, got it

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java (hunk 2 above)
Comment (donikrizky) [reply src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java]: wait, it is in the outdated version haha, I've changed it

**Pull request merged**